web/wiki/src/__tests__/organisms/CardResource.test.tsx

```diff
@@ -1,6 +1,6 @@
 import { vi } from 'vitest'
 import { render, screen } from '../test-utils'
-import CardResource from '../../components/organisms/CardResource'
+import { CardResource } from '../../components/organisms'
 import { TAuthContext, useAuth } from '../../context/AuthProvider'
 import { TCardResource } from '../../types'
 
```

Comment:
Please, include tests for new code from VoteCounter

Reply:
It will be addressed in issue #1060